common/types.go

```diff
@@ -218,3 +269,583 @@ func (b *BidTraceV2WithTimestampJSON) ToCSVRecord() []string {
 		fmt.Sprint(b.TimestampMs),
 	}
 }
+
+type SignedBeaconBlindedBlock struct {
```

Comment:
It has a typo, I think. Shouldn't it be called `SignedBlindedBeaconBlock`?

Also, should this perhaps go into the attestant types? (no rush)